test/tools/libtesteth/Options.cpp

```diff
@@ -59,6 +59,7 @@ void printHelp()
 	cout << setw(30) << "--randomcode <MaxOpcodeNum>" << setw(25) << "Generate smart random EVM code\n";
 	cout << setw(30) << "--createRandomTest" << setw(25) << "Create random test and output it to the console\n";
 	cout << setw(30) << "--seed <uint>" << setw(25) << "Define a seed for random test\n";
+	cout << setw(30) << "--options <PathTo.json>" << setw(25) << "Use following options file for random code generation\n";
```

Comment:
The name `--options` is too generic.  Maybe `--randomCodeOptions`?

Reply:
you see it will only be used with --createRandomTest  so I tried to keep it short

Reply:
`--createRandomTestOptions` is better than `--options`.